### ISSUE TITLE
loss = nan       |g| = nan

### ISSUE THREAD

**Issue by zswartz1993** [melodyguan/enas#36]
I am attempting to run the cifar10 macro search on a set of images that I have converted into the same format as for cifar10. During the child training phase, the loss and |g| are always "nan," because the scripts are expecting only 10 classes, while I am using many more. Does anyone know where in the cifar10 scripts I can specify the number of classes?
